.github/workflows/nightly_tests.yaml

```diff
@@ -162,6 +164,33 @@ jobs:
       if: always()
       run: python xpk.py cluster delete --cluster $PATHWAYS_TPU_CLUSTER_NAME --zone=us-central2-b
 
+  rc-cluster:
```

Comment:
Can we have at least workflow run and checked for execution status to ensure this is usable? Also, if you modify niggtly tests, please run them before merging and attach results to task description. If you are not able to run them please let me know.

Reply:
how do I run the nightly tests? Also run which workflow?

Reply:
In actions select nightly tests and run from branch